### DISCUSSION TITLE
20211103B-update-special-geometry.ts is just a placeholder?

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15478] [Archive]
![image](https://user-images.githubusercontent.com/8686739/189091353-45081b7f-5d5d-4c2c-b4d1-8af9b23bb227.png)

what is meaning?

**Comment by azrikahar** [comment]
Understandably it's not as obvious at a quick glance, but the REPLACE gives a hint on what it's doing, which is switching commas to periods, and vice versa 🙂

![image](https://user-images.githubusercontent.com/42867097/189100493-d4d51716-162a-4207-8505-94ce1233025f.png)

**Reply by anlexN** [reply]
thank you.
